.gitignore

```diff
@@ -8,3 +8,4 @@
 /node_modules
 /test/tmp
 tsconfig.tsbuildinfo
+.local
```

Comment:
What is this?

Reply:
oh, I had to use a bunch of secrets/tempfiles for local testing and must've added the folder I put them in gitignore... I'll remove this line